service/src/main/java/uk/gov/hmcts/reform/fpl/model/docmosis/DocmosisNoticeOfProceeding.java

```diff
@@ -18,8 +19,6 @@
     private final String legalAdvisorName;
```

Comment:
Do you wanna extract this and the one above into the already existent DocmosisJudgeAndLegalAdvisor?

Reply:
Yeah that's a good idea